index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>subrat&ridwan</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+  <div id="container">
+    <div id="header">
+       <h1 id="title">freeCodeCamp Survey Form Survey Form</h1>
+       <p id="description"><em>Thank you for taking the time to help us improve the platform</em></p>
+    </div>
+ 
+
+  <form id="survey-form">
+    <div class="form-control">
+      <label id="name-label"><strong>Name</strong></label>
+      <input class="input" type="text" placeholder="Enter your name" name="name" id="name"  required>
+    </div>
+    <div class="form-control">
+      <label id="email-label"><Strong>Email</Strong></label>
+      <input  class="input"  type="email" placeholder="Enter your email"  name="email" id="email"  required>
+    </div>
+    <div class="form-control">
+      <label id="number-label"><strong>Age (optional)</strong></label>
+      <input  class="input"  type="number" placeholder="Enter your age"  name="age" id="number" min="10" max="100" required>
+    </div>
+    <div class="form-control">
+      <label for="Dropdown">Dropdown options:</label>
+      <select name="Dropdown" id="dropdown">
+        <option disabled="" selected="" value="">Select an option</option>
+        <option value="option1">Dropdown Option 1</option>
+        <option value="option2">Dropdown Option 2</option>
+        <option value="option3">Dropdown Option 3</option>
+        <option value="option4">Dropdown Option 4</option>
+      </select>  
+    </div>  
```

Comment:
- [ ] **Required** When giving a code review, it's better to use of [GitHub inline comments](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) rather than summarize. Check the link out for more about inline comments. This points the student to the place in the code where the suggested correction should be effected.
- [ ] **Required** Rather than give orders and instructions, give suggestions and use emojis and gifs liberally to convey your points. This gives a warm reception from the student. I hope this makes sense 😄😄.
- [ ] **Required** You should try to give examples and other forms of explanation as to why a change is needed. The student will not only implement the imminent change but learn why it's necessary and thus become a better programmer.